int16[3] colorA
int16[3] colorB 

# colorA = [red, green, blue] = [0..255, 0..255, 0..255]
# colorB = [red, green, blue] = [0..255, 0..255, 0..255]
---
int8 value 
# Return values:
#  0    := Success
# -1    := invalid value